Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -2022,6 +2027,17 @@ def sophos_central_endpoint_list_command(client: Client, args: dict) -> CommandR
     if items:
         for item in items:
             object_data = {field: item.get(field) for field in table_headers}
+            # The Sophos API does partial string matching, which can be unwanted for IP addresses.
+            if full_match_ip:
+                query_ips = argToList(args.get("ip_addresses"))
+                ips = []
+                if (data := object_data.get("ipv4Addresses", [])) is not None:
+                    ips.extend(data)
+                if (data := object_data.get("ipv6Addresses", [])) is not None:
+                    ips.extend(data)
```

Comment:
Same as comment above:
```suggestion
                ips.extend(object_data.get("ipv6Addresses", []))
```

Reply:
Same as my comment above.